### ISSUE TITLE
Community Partner section needs a Update

### ISSUE THREAD

**Issue by imanishbarnwal** [hack4bengal/hack4bengal.github.io#56]
### Description

In the community partner scection, there are two rows, and one of them is fixed also some logos are repeated which is not look good. Let's make all of them in one row or anything better you can do, and allow them to move as it is also keep an arrow on both sides so that users can change or view the logo. In the mobile version, it looks good but if you can suggest some improvements are always welcome. Thank You!!

### Screenshots

![image](https://user-images.githubusercontent.com/46371923/194085221-1708f430-a19a-4f55-a1a3-1103b7cda26c.png)

### Additional information

Before sending PR kindly share some screenshots. Thank You :)

**Comment by andreaevolo** [comment]
If I understood crearly the logos should be in one 1 row and the user should be able to scroll them like in a carousel.
Am I right?

**Comment by imanishbarnwal** [comment]
Yes, but we can also use two then just make sure the logos should not be repeated in one go.

**Comment by YuriDevAT** [comment]
Hi, I would like to work on this issue.

**Comment by imanishbarnwal** [comment]
Hi @YuriDevAT please go ahead. Assigned you :)
